developer-docs-site/docs/sdks/ts-sdk/index.md

```diff
@@ -6,6 +6,6 @@ hidden: false
 
 # TS SDK
 
-- ### [Typescript SDK](typescript-sdk)
-- ### [Typescript SDK Overview](typescript-sdk-overview)
-
+- ### [Typescript SDK Installation](typescript-sdk.md)
+- ### [Typescript SDK Overview](typescript-sdk-overview.md)
+- ### [Typescript SDK Reference](https://aptos-labs.github.io/ts-sdk-doc/)
```

Comment:
This is deprecated. We don't reference this link anymore. There are still some references to this link in SDK examples and I am going to remove those as well.

Reply:
> This is deprecated. We don't reference this link anymore. There are still some references to this link in SDK examples and I am going to remove those as well.

I understand and have removed the line. Please let me know if the other tiny change is worthwhile here.